bank/persistence/mongo.go

```diff
@@ -83,7 +83,7 @@ func (se *mongoSessionStore) GetAccounts(userID string) ([]*domain.Account, erro
 	session := se.Session.Clone()
 	defer session.Close()
 
-	var result []*domain.Account
+	result := make([]*domain.Account, 0)
```

Comment:
Make sure that your brach is up to date with upstream as this change is already there and should not be in this PR.